packages/building-blocks/package.json

```diff
@@ -12,7 +12,7 @@
     "mapbox-gl": "npm:empty-npm-package@1.0.0",
     "maplibre-gl": "^2.1.9",
     "react-map-gl": "^7.0.15",
-    "@opentripplanner/core-utils": "9.0.3"
+    "@opentripplanner/core-utils": "11.1.2"
```

Comment:
For a next time, try to not include changes to unrelated packages in PRs.

Reply:
This is just here because it was causing the old core-utils version to show up in yarn.lock!